content/prerequisites/workspaceiam.md

```diff
@@ -28,3 +28,31 @@ echo "export AWS_REGION=${AWS_REGION}" >> ~/.bash_profile
 aws configure set default.region ${AWS_REGION}
 aws configure get default.region
 ```
+
+### Validate the IAM role
+
+Use the [GetCallerIdentity](https://docs.aws.amazon.com/cli/latest/reference/sts/get-caller-identity.html) CLI command to validate that the Cloud9 IDE is using the correct IAM role.
+
+#### VALID
+
+If the _Arn_ contains `modernizer-workshop-cl9` and an Instance ID, you may proceed.
+
+```output
+{
+    "Account": "123456789012", 
+    "UserId": "AROA1SAMPLEAWSIAMROLE:i-01234567890abcdef", 
+    "Arn": "arn:aws:sts::123456789012:assumed-role/modernizer-workshop-cl9/i-01234567890abcdef"
```

Comment:
Can we provide alternative way to get correct info by providing steps similar to this link: https://eksworkshop.com/pr-195/logging/prereqs/
This way the user can validate the role attached to Cloud9 is the same as get-caller-identity call